### DISCUSSION TITLE
Accessory Automation - Home Mode Option

### DISCUSSION THREAD

**Discussion by melvin-reyes** [dgreif/ring#1544] [General]
@dgreif Thanks so much for your work on this plugin it's amazing!

I was wondering if there was a way to have a home mode when setting up an automation. Currently, I can only choose one of the following: Triggers, Disarms and Arms. In my head I feel like this is a limitation with HomeKit and not the plugin but figured I would ask. 

Thanks in advance.

**Comment by dgreif** [comment]
Unfortunately the plugin has no control over what options are exposed for automation within HomeKit. Would need to submit a feature request to Apple to get it added there.
